### PR TITLE
Don't capitalize acronym strings

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Tweak the JS capitalize() function to not capitalize strings only
composed of capital letters and dashes / underscores since those are
probably acronyms. An example would be SR-IOV: we don't want such a
formula name to be capitalized.

## GUI diff

Before:
Screenshot_2021-08-04 Uyuni - Systems(1)

After:
Screenshot_2021-08-04 Uyuni - Systems(2)

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: Harmless UI label change

- [X] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by Etheryte — CHANGES_REQUESTED** [review]
It would be nice to add some unit tests to this, just a case out of every category of a-la acronym, non-acronym, and edge cases would be good. Right now it isn't clear whether this code works correctly or not (since I don't know what the input values are expected to be), but it _looks_ like it might have bugs. For example, the regex matches any string that contains a capital letter or a dash, even if it isn't comprised solely of them.  

Suggested test cases:  
 - `"foo"`
 - `"FOO"`
 - `"FO-O"`
 - `"fooBar"`
 - `"foo-bar"`
 - anything else I might've missed?

**Comment by cbosdo** [comment]
OK I'll add those cases and try to improve the regexp. I don't think there are other cases to be covered

**Review by Etheryte — APPROVED** [review]
Looks good, thanks a lot. 👍

File: web/html/src/utils/functions.test.ts [hunk 1]
```diff
@@ -28,6 +28,18 @@ test("check order by date function'", () => {
   expect(arr[0]).toEqual(eRaw);
 });
 
+describe("capitalize", () => {
+  test("simple cases", () => {
+    expect(Utils.capitalize("foo")).toEqual("Foo");
+    expect(Utils.capitalize("foo-bar")).toEqual("Foo Bar");
+    expect(Utils.capitalize("fooBar")).toEqual("Foobar");
```
Comment (ncounter) [inline web/html/src/utils/functions.test.ts:35]: If I may, I would actually expect `fooBar` to became `Foo Bar`, but it's just my feeling. Do you have instead any concrete example where your rule applies properly?

Comment (cbosdo) [reply web/html/src/utils/functions.test.ts:35]: `fooBar` converted to `Foobar` is the previous behavior... I didn't change that at all.

Comment (ncounter) [reply web/html/src/utils/functions.test.ts:35]: Okay then, never mind it's not part of this PR. Thanks for clarifying :+1:

**Review by ncounter — APPROVED** [review]
